### ISSUE TITLE
❌ Software Removal | 

### ISSUE THREAD

**Issue by oldphones68** [privacytools/privacytools.io#2245]
## Description
The homepage of pt.io contains "The Ultimate Guide to Online Privacy - Comprehensive "Ninja Privacy Tips" and 150+ tools."
But, from visiting site, I found that site is not getting updated for a long time.

Besides, this site is also recommending Ghostery and AdBlockPlus.

These are NOT acceptable because Ghostery has turned into an ad company and ABP 'Allows acceptable ads'!

## Why I am making the suggestion
Mentioned above. 

## My connection with the software
Nothing, ex-user of the softwares mention above.

No

I will keep the issue up-to-date if something I have said changes or I remember a connection with the software.

**Comment by Dyrimon** [comment]
I agree, the site is just shilling VPNs left and right. Affiliate links galore!

**Comment by freddy-m** [comment]
I think this can be dealt with in #1984

**Comment by dngray** [comment]
Agreed we need to be removing this.
